Detections/InfobloxNIOS/ExcessiveNXDOMAINDNSQueries.yaml

```diff
@@ -22,7 +22,7 @@ query: |
   let threshold = 200;
   InfobloxNIOS
   | where TimeGenerated >= timeframe
```

Comment:
change 
`let timeframe = 1h`
and condition to 
`| where TimeGenerated > = ago(timeframe)`

Reply:
Revisions have been made